src/app.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import * as ReactDOM from 'react-dom/client';
+import { store } from './app/store';
+import { Provider } from 'react-redux';
+import CodeInputAndController from './features/code-input-and-control/code-input-and-control';
+import './features/console/console.module'
+import CustomConsole from './features/console/console.module';
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
+
+root.render((
+  <React.StrictMode>
+    <Provider store={store}>
```

Comment:
Wenn du deine 4 CodeInputController nochmal in ein div wrappst, kannst du das Layouting viel eleganter mit  einem CSS grid machen https://css-tricks.com/snippets/css/complete-guide-grid/#aa-introduction. Wenn du richtig fancy bist, kannst du das damit sogar auf CSS ebene soweit konfigurieren, dass der nutzer selber eingeben kann, wieviele Editoren er möchte und die damit automagisch layouten :slightly_smiling_face: